slidez-fe-core/src/containers/session/store/responseHandler.ts

```diff
@@ -21,7 +24,13 @@ function throwBadType(type: string): never {
 export const responseHandler =
     (dispatch: any) => (response: GenericResponse<SessionResponse, string>) => {
         if (response.error || !response.data) {
-            throw new Error(response.error || 'No data')
+            handleNotification(
+                'Error',
+                'Link doesn`t exist',
+                NotificationTypes.ERROR
+            )
+            window.location.assign(`/#${AppRoute.EVENTS}`)
```

Comment:
Log this exception please
console.error(response.error)

Reply:
add this, thanks)